holmes/plugins/toolsets/datetime.py

```diff
@@ -0,0 +1,35 @@
+from holmes.core.tools import ToolsetTag
+from typing import Dict
+from holmes.core.tools import Tool, Toolset
+import datetime
+
+
+class CurrentTime(Tool):
+    def __init__(self):
+        super().__init__(
+            name="get_current_time",
+            description="Return current time information. Useful to build queries that require a time information",
+            parameters={},
+        )
+
+    def _invoke(self, params: Dict) -> str:
+        now = datetime.datetime.now(datetime.timezone.utc)
+        return f"The current UTC date and time are {now}. The current UTC timestamp in seconds is {int(now.timestamp())}."
+
+    def get_parameterized_one_liner(self, params) -> str:
+        return "fetched current time"
+
+
+class DatetimeToolset(Toolset):
```

Comment:
Thinking here loud.
A. Do we want to make sure that this toolset always enabled and presented? if so let's open a tech debt ticket to make some tools immutalbe.
B. Are we fine to show it in the toolsets screen in the platform?

Reply:
I would not add an override for such tools b/c I don't see the value. Same for showing it. If it's a tool, it should be shown like the other tools.